lib/cartopy/mpl/geoaxes.py

```diff
@@ -49,7 +49,7 @@
 import cartopy.mpl.patch as cpatch
 from cartopy.mpl.slippy_image_artist import SlippyImageArtist
 from cartopy.vector_transform import vector_scalar_to_grid
-
+from cartopy import geodesic
```

Comment:
Ah - my bad. Should just be `geodesic.Geodesic`.

Just a thought - does it make more sense to put the import within the tissot method, so that it isn't required for geoaxes to work as discussed in #665?